tests/printer/comments/expected/expr.res.txt

```diff
@@ -155,10 +153,8 @@ let distXToTraverse = if deltaX < 0. {
   } else {
     deltaX +. minRangeX
   }
-} else if (
-  // wizard to the right
-  minRangeX >= deltaX
-) {
+} // wizard to the right
```

Comment:
The printed code doesn't look that "smooth".

Can you log a separate issue to check if we can preserve nested if else branches?
```
if a {
  b
} else {
  // test
  if c {
    d
  }
}

```

Reply:
I'll log an issue for tracking, but I actually thought the flattening was a feature here. Of course, if flattening, it is a bit of an edge case that a comment is between the else and the if. The user can still move the comment down manually after formatting in this case.

Reply:
Maybe it's only a feature when we don't have comments? we can think about it later